modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -87,52 +80,46 @@ If a file is no longer needed, you can remove it from the server by clicking the
 After removing a file from the server, you also need to manually remove data mapping using that file.
 Otherwise, a "file not on server" error will be triggered when loading data.
 
-Continue with <<Map data files to vertex type or edge type>>.
-
-=== Create S3 data source
-
+--
+Amazon S3::
++
+--
 TigerGraph creates a connection to S3 in order to access the file system in an S3 bucket.
 Only one file may be selected at a time.
 
 [NOTE]
-====
 Data files, after decompression, must be in either CSV or Parquet format.
 If a file is in Parquet format, the data you are loading cannot use INT96 data types.
-Date in INT96 types are not loaded into TigerGraph and the respective attribute will be left empty. 
 
 TigerGraph supports loading from archived and compressed S3 files directly.
 Currently supported file extensions include zip, tar.gz, tgz and tar.
 GraphStudio detects the file extension and automatically chooses the corresponding file format.
 If the file is encoded with one of these formats but has a non-standard file extension, you can manually specify the file format.
-====
 
 
-. After you click the btn:[S3] data source icon, you will be prompted to first connect an S3 data source.
-+
-image::choose-file-from-s3-bucket.png[]
+. After you click the btn:[S3] data source icon, you will be prompted for your:
 
-. Click the Add new data source button image:upload_file_btn.png[] to bring up the new S3 data source window.
-Give a name to the data source and provide the access key id and secret access key to connect to S3.
-+
-image::new-s3-data-source.png[]
-. Click the btn:[ADD] button and the data source will appear in the Data Source list.
+* AWS access key ID
+* AWS secret access key
 
-. Clicking the data source in the list expands a file tree listing all buckets and folders accessible by the credentials provided.
+You also have the option to add an alias to the data source connection.
 
-. Choose the file you want to add and change the parsing options if necessary. The first ten lines of the selected file appear as a preview.
-+
-image::Screen Shot 2019-05-16 at 12.56.32 PM.png[]
 
-. Confirm that the data has parsed correctly (see <<Configure the File Parser>>) and click btn:[ADD] to add the data source to your project, where it will appear in the working panel.
-. Continue with <<Map data files to vertex type or edge type>>.
+. Identify your file as CSV or JSON format.
 
-=== Create a GCS data source
+. Enter the S3 URI.
 
+. Confirm that the data has parsed correctly and click btn:[ADD] to add the data source to your project, where it will appear in the working panel.
```

Comment:
```suggestion
. Confirm that the data has parsed correctly and click btn:[ADD] to add the data file to your project, where it will appear in the working panel.
```